lib/Steps/Integrations/NextJs.ts

```diff
@@ -144,20 +145,17 @@ export class NextJs extends BaseIntegration {
   }
 
   private _checkDep(packageName: string, minVersion?: boolean): boolean {
-    const depVersion = parseInt(
-      _.get(appPackage, ['dependencies', packageName], '0').replace(/\D+/g, ''),
-      10,
+    const depVersion = _.get(
+      appPackage,
+      ['dependencies', packageName],
+      '0.0.0',
     );
-    const devDepVersion = parseInt(
-      _.get(appPackage, ['devDependencies', packageName], '0').replace(
-        /\D+/g,
-        '',
-      ),
-      10,
+    const devDepVersion = _.get(
+      appPackage,
+      ['devDependencies', packageName],
```

Comment:
Why do we check `devDeps`? Next should always be a dependency, if it's not something is wrong.

Reply:
Yes, it should. But having it as a dev dependency runs the project successfully, and users might have it there. I rather check it, than not doing it and having the chance of getting issues because of it.